docs/concepts/components.md

```diff
@@ -4,7 +4,9 @@ description: Components and their lifecycle hooks
 ---
 ## What are Components?
 
-Components are the building blocks of Yew. They manage their own state and can render themselves to the DOM. Components are created by implementing the `Component` trait which describes the lifecycle of a component.
+Components are the building blocks of Yew. They manage their own state and can render themselves to the DOM. Components are created by implementing the `Component` trait on object (e.g. structs or enums). The `Component`
```

Comment:
To stay consistent with "the book" <sup>[[source]](https://doc.rust-lang.org/book/ch10-02-traits.html#implementing-a-trait-on-a-type)</sup>:
```suggestion
Components are the building blocks of Yew. They manage their own state and can render themselves to the DOM. Components are created by implementing the `Component` trait for a type. The `Component`
```

Reply:
That's definitely clearer.